web/client/plugins/MousePosition.jsx

```diff
@@ -64,11 +64,22 @@ const MousePositionButton = connect((state) => ({
 
 const MousePositionComponent = require('../components/mapcontrols/mouseposition/MousePosition');
 
-
 class MousePosition extends React.Component {
+
```

Comment:
add property degreesTemplateStr with a default (this will be the one that will be overwritten from localConfig
```
static propTypes = {
        degreesTemplateStr: PropTypes.string
    };
    static defaultProps = {
        degreesTemplateStr: "MousePositionLabelDMS"
    };
```